Task_functions/index.js

```diff
@@ -1,11 +1,15 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-
-
-
-//Write code for the 2nd task below this
-
-
+//Write code for the 1st task below this
+function add(num1, num2){

```

Comment:
It is better to use template literals for string interpolation instead of concatenation. e.g., `${greeting} ${name}`